handler/handler.go

```diff
@@ -166,11 +166,11 @@ func (h *Handler) saveAndReturnAnnotations(ctx context.Context, w http.ResponseW
 	}
 
 	w.Header().Set(annotations.DocumentHashHeader, newHash)
```

Comment:
Do we still want to set the header in the response?

Reply:
I think so. We only delete the body.